src/torrent.cpp

```diff
@@ -1492,7 +1510,7 @@ bool is_downloading_state(int const st)
 			m_save_path,
 			static_cast<storage_mode_t>(m_storage_mode),
 			m_file_priority,
-			m_info_hash
+			m_info_hash.has_v1() ? m_info_hash.v1 : m_info_hash.get(protocol_version::V2)
```

Comment:
this basically looks like "get_worst()". would it not work to just use `get_best()` instead?

Reply:
I was probably thinking of backwards compatibility when I wrote this. In hindsight that's not an issue here since hybrid torrents will have a different v1 info-hash from an equivalent v1 only torrent.